soh/src/code/z_parameter.c

```diff
@@ -2181,6 +2181,14 @@ u16 Randomizer_Item_Give(GlobalContext* globalCtx, GetItemEntry giEntry) {
         return RG_NONE;
     }
 
+    if (item == RG_MAGIC_BEAN_PACK) {
+        if (gSaveContext.inventory.items[SLOT(ITEM_BEAN)] == ITEM_NONE) {
```

Comment:
can we just do `INV_CONTENT(ITEM_BEAN) == ITEM_NONE` here?

Reply:
Maybe, I'll try it out. I'm pretty sure I copied that from the vanilla code.

Reply:
It does work, resolved in 53531c4481123e448f731f01d5152ac0bd5cf049